src/events/logs.py

```diff
@@ -78,6 +83,8 @@ def get_regex_list():
     regex.append([events.on_fail_delete_flowcontrollers,
                   r"PRESParticipant_destroyOneFlowControllerWithCursor:" +
                   r"has (\d+) writers on flowcontroller"])
+    regex.append([events.on_inconsistent_transport_discovery_configuration,
+                  r"\[D0000\|ENABLE\]DDS_DomainParticipant_enableI:Automatic participant index failed to initialize\. PLEASE VERIFY CONSISTENT TRANSPORT \/ DISCOVERY CONFIGURATION\."])
```

Comment:
Remove the context `[D0000|ENABLE]` from the regular expression. They could not appear in some situations.